src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import {
+  Grid,
+  Avatar as AvatarMaterialUI,
+  makeStyles,
+  CircularProgress,
+} from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+import { AvatarProps } from './types';
+
+const useRegularTypeStyles = makeStyles({
+  avatar: {
+    margin: 10,
+    width: 60,
+    height: 60,
+    marginBottom: 20,
+  },
+  name: {
+    fontSize: 16,
+    fontWeight: 700,
+    lineHeight: 1.2,
+  },
+  subtitle: {
+    fontSize: 12,
+  },
+  progress: {
+    position: 'absolute',
+    top: -5,
+  },
+  grid: {
+    position: 'relative',
+  },
+});
+
+const useSmallTypeStyles = makeStyles({
+  avatar: {
+    margin: 10,
+    width: 40,
+    height: 40,
+    marginBottom: 20,
+  },
+  name: {
+    fontSize: 14,
```

Comment:
@michaelmoney you should not define the font-size since it's already defined in the default theme [default theme](https://material-ui.com/customization/default-theme/#default-theme) you can expand it and find the default `rem` values.

Reply:
Fixed: https://github.com/BuildIt-Poland/buddy-app/pull/18/commits/81f2a972b03b57fabeb5645aa03da2fdbc068ab2

Reply:
We can archive that just by using `variant` and `component` attributes.
For example if we can to have 1rem we can simple use `Typography variant={'body1'}`

Reply:
And we can delete extra styles of `font-size`